kafka-connect-s3/src/main/java/io/confluent/connect/s3/hooks/BlockingKafkaPostCommitHook.java

```diff
@@ -109,12 +99,19 @@ public void put(Set<String> s3ObjectPaths, Long baseRecordTimestamp) {
     }
   }
 
-  private String roundTimeToPartitonTime(Long baseRecordTimestamp) {
+  private String getLocalDateTime(String s3ObjectPath, Long baseRecordTimestamp) {
+    if (baseRecordTimestamp == null) {
+      return null;
+    }
     LocalDateTime localDateTime = LocalDateTime.ofInstant(
             Instant.ofEpochMilli(baseRecordTimestamp), ZoneOffset.UTC);
-    LocalDateTime roundedDateTime = localDateTime.truncatedTo(ChronoUnit.HOURS).plusMinutes(
-            (localDateTime.getMinute() / partitionDurationMinutes) * partitionDurationMinutes);
-    return roundedDateTime.format(timeFormatter);
+
+    log.info("for {}: localtime of {} is {}"
```

Comment:
This probably should be debug, or debug after several iterations when we see it's "stable"
and I would change a message itself

Reply:
yes, it was a debug message, will fix wording and change level to debug